README.md

```diff
@@ -1,51 +1,36 @@
-# cluster-registry
+# Cluster Registry
 
-This repository contains the code for the cluster registry. The cluster registry
-is an effort being led under the auspices of sig-multicluster.
+A lightweight tool for maintaining a list of clusters and associated metadata.
 
-This is currently in the prototyping stage, and is not yet ready for use except
-by contributors and experimenters.
+# What is it?
 
-If you have questions, please reach out to
-[kubernetes-sig-federation](https://groups.google.com/forum/#!forum/kubernetes-sig-federation).
+The cluster registry helps you keep track of and perform operations on all of
+your clusters. This repository contains an implementation of the cluster API=
```

Comment:
`=`?

Reply:
That's not supposed to be there! Removed.